src/clj_sqs_extended/aws/sqs.clj

```diff
@@ -69,6 +85,7 @@
 
   ([sqs-client queue-name
     {:keys [fifo
+            visibility-timeout-in-seconds
             kms-master-key-id
             kms-data-key-reuse-period]
      :or   {fifo true}
```

Comment:
`fifo` should not be an argument. `assoc` `fifo=true` into opts argument to ensure that it's true.